addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/handler/KodiHandler.java

```diff
@@ -193,13 +195,29 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     connection.updatePlayerStatus();
                 }
                 break;
+            case CHANNEL_THUMBNAIL:
```

Comment:
This is exactly what came into my mind when I first looked over the code, but finally I forgot to refactor it.